File: 백준/Bronze/10951. A＋B － 4/A＋B － 4.js
let input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n");

for (let i = 0; i < input.length; i++) {
  let [A, B] = input[i].split(" ");
  result = parseInt(A) + parseInt(B);
  console.log(result);
}

File: 백준/Bronze/10951. A＋B － 4/A＋B － 4.kt
fun main() {
    val result = mutableListOf<Int>()

    while (true) {
        val input = readlnOrNull()
        if (input == null) break
        if (input != null) {
            val (A, B) = input.split(" ").map { it.toInt() }
            result.add(A + B)
        }
    }

    println(result.joinToString("\n"))
}